ros2bag/ros2bag/verb/play.py

```diff
@@ -42,6 +42,9 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             '-r', '--rate', type=check_positive_float, default=1.0,
             help='rate at which to play back messages. Valid range > 0.0.')
         parser.add_argument(
+            '--topics', type=str, default='', nargs='*',
```

Comment:
shouldn't that be `nargs='+'`? Otherwise I could call `ros2 bag play --topics` and don't specify any topics. I think that's somewhat weird.

Reply:
520fd60